testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -61,31 +62,9 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64" product has been added
     Then the SLE15 SP4 product should be added
 
-@scc_credentials
 @uyuni
-  Scenario: Add SLES 15 SP4 product with recommended sub-products
-    When I follow the left menu "Admin > Setup Wizard > Products"
-    And I wait until I do not see "Loading" text
-    And I enter "SUSE Linux Enterprise Server 15 SP4" as the filtered product description
-    And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64" text
-    And I open the sub-list of the product "SUSE Linux Enterprise Server 15 SP4 x86_64"
-    And I open the sub-list of the product "Basesystem Module 15 SP4 x86_64"
-    And I open the sub-list of the product "Desktop Applications Module 15 SP4 x86_64"
-    Then I should see that the "Basesystem Module 15 SP4 x86_64" product is "recommended"
-    And I should see that the "Server Applications Module 15 SP4 x86_64" product is "recommended"
-    When I select "SUSE Linux Enterprise Server 15 SP4 x86_64" as a product
-    Then I should see the "SUSE Linux Enterprise Server 15 SP4 x86_64" selected
-    And I should see the "Basesystem Module 15 SP4 x86_64" selected
-    And I should see the "Server Applications Module 15 SP4 x86_64" selected
-    When I select "Desktop Applications Module 15 SP4 x86_64" as a product
-    And I select "Development Tools Module 15 SP4 x86_64" as a product
-    Then I should see the "Desktop Applications Module 15 SP4 x86_64" selected
-    And I should see the "Development Tools Module 15 SP4 x86_64" selected
-    When I select "Containers Module 15 SP4 x86_64" as a product
-    Then I should see the "Containers Module 15 SP4 x86_64" selected
-    When I click the Add Product button
-    And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64" product has been added
-    Then the SLE15 SP4 product should be added
+  Scenario: Add openSUSE Leap 15.4 product, inlcuding Uyuni Client Tools
+    When I use spacewalk-common-channel to add channel "opensuse_leap15_4 opensuse_leap15_4-non-oss opensuse_leap15_4-non-oss-updates opensuse_leap15_4-updates opensuse_leap15_4-backports-updates opensuse_leap15_4-sle-updates uyuni-proxy-devel-leap opensuse_leap15_4-uyuni-client" with arch "x86_64"
```

Comment:
```suggestion
    When I use spacewalk-common-channel to add channels with arch "x86_64":
    | opensuse_leap15_4                   |
    | opensuse_leap15_4-non-oss           |
    | opensuse_leap15_4-non-oss-updates   |
    | opensuse_leap15_4-updates           |
    | opensuse_leap15_4-backports-updates |
    | opensuse_leap15_4-sle-updates       |
    | uyuni-proxy-devel-leap              |
    | opensuse_leap15_4-uyuni-client      |
```


```
When(/^I use spacewalk\-common\-channel to add channels with arch "([^"]*)":$/) do |arch, channels|
  child_channels = channels.hashes.arr.join(' ')
  command = "spacewalk-common-channels -u admin -p admin -a #{arch} #{child_channels}"
  $command_output, _code = get_target('server').run(command)
end
```

Reply:
Using a [data table](https://cucumber.io/docs/gherkin/reference/#data-table) here makes indeed sense and improves the readability.

Reply:
Since 

```ruby
child_channels = channels.hashes.arr.join(' ')
```

```bash
Scenario: Add openSUSE Leap 15.4 product, inlcuding Uyuni Client Tools    # features/core/dom.feature:5
      This scenario ran at: 2023-08-03 10:51:32 +0200
    When I use spacewalk-common-channel to add channels with arch "x86_64": # features/step_definitions/command_steps.rb:136
      | opensuse_leap15_4                   |
      | opensuse_leap15_4-non-oss           |
      | opensuse_leap15_4-non-oss-updates   |
      | opensuse_leap15_4-updates           |
      | opensuse_leap15_4-backports-updates |
      | opensuse_leap15_4-sle-updates       |
      | uyuni-proxy-devel-leap              |
      | opensuse_leap15_4-uyuni-client      |
      undefined method `arr' for #<Array:0x0000556c37fb99a8> (NoMethodError)
      ./features/step_definitions/command_steps.rb:137:in `/^I use spacewalk\-common\-channel to add channels with arch "([^"]*)":$/'
      features/core/dom.feature:6:in `I use spacewalk-common-channel to add channels with arch "x86_64":'
```
does not work, I will leave this unchanged and we can tackle this in a future PR, where we will have a look at other places with long lines and do everything in one PR.